spot-client/src/common/app-state/setup/selectors.js

```diff
@@ -20,6 +20,16 @@ export function getDisplayName(state) {
     return state.setup.displayName;
 }
 
+/**
+ * Returns the JWT token for authentication with backend services.
```

Comment:
I'm a smug jerk who likes pointing this out: JWT token -> JWT

Reply:
do you want me to also rename all occurrences of `jwtToken` to just `jwt` ?

Reply:
Now that you've pointed it out, yes.

Reply:
done